.travis.yml

```diff
@@ -30,7 +30,6 @@ matrix:
     - php: 5.6
       env: SYMFONY_VERSION=3.0.*
   allow_failures:
-    - php: nightly
```

Comment:
hmm not sure .. I guess we keep 2.8/3.0 in allowed failures too .. then again .. it doesn't help in spotting issues .. maybe we should only have it in allowed failures in case there are problems .. ie. when we find issues we add it to allowed failures and report the bug and then remove it again as soon as possible.